fire/core.py

```diff
@@ -166,13 +171,18 @@ class FireExit(SystemExit):
 
   Contains the trace of the Fire program.
 
-  If not caught, then a FireExit will cause the program to exit with a non-zero
-  status code.
+  Note:
+    Since this exception inherits from SystemExit/BaseException, you'll have to
+    explicitly catch it with `except SystemExit` or `except FireExit`.
   """
-
-  def __init__(self, status, data):
-    super(FireExit, self).__init__(status)
-    self.trace = data
+  def __init__(self, code, trace):
```

Comment:
Let's avoid shadowing things from the global namespace.
How about we do `self.trace = component_trace` so that we're not introducing yet another name for traces.

Also can you link to the pylintrc you found?